docs/en/command/ddl.md

```diff
@@ -227,13 +227,13 @@ use Yiisoft\Db\Connection\ConnectionInterface;
 
 /** @var ConnectionInterface $db */
 $db->createCommand()->addForeignKey(
-    '{{%customer}}',
-    'fk-customer-profile_id',
-    'profile_id',
-    '{{%profile}}',
-    'id',
-    'CASCADE',
-    'CASCADE'
+    table: '{{%customer}}',
```

Comment:
o'm not like named params

Reply:
In this case we can provide it as is w/o named arguments.

Reply:
@darkdef Why? In case of a bigger amount of arguments / when it's hard to tell by values what the argument names are, it's useful and convenient.

@samdark Surely we can, but it improves readability in this case, no need to look for method signature / description.

They also can be applied partially.

Reply:
In this particular example, it's hard to differentiate between `table` / `refTable`, `columns` / `refColumns` and `delete` / `update`.

Reply:
Reverted this part. Will decide if we want it after release.